releasenotes/notes/unify-instructions-and-commands-aaa6d8724b8a29d3.yaml

```diff
@@ -22,6 +22,8 @@ features:
 
         sched += Delay(5, DriveChannel(0))
         sched += ShiftPhase(np.pi, DriveChannel(0))
+        sched += SetFrequency(5.5, DriveChannel(0))
+
 deprecations:
   - |
     ``DelayInstruction`` has been deprecated and replaced by
```

Comment:
I think this one might also deserve a `feature` release note in addition to this line here. This is a new capability that we didn't have before

Reply:
Sure, where do we need to write that?

Reply:
The best is probably a new note: `reno new <short description, for example: add-set-frequency-instruction>`. If you don't have reno, then `pip install reno` in your env. Then it'll autogenerate a reno note in the right place, and you can delete all the sections except `feature`